pkg/common/config/kafkaconfigmapwatcher.go

```diff
@@ -35,13 +35,7 @@ type LoggingObserver func(ctx context.Context, configMap *corev1.ConfigMap)
 // Initialize The Specified Context With A ConfigMap Watcher
 // Much Of This Function Is Taken From The knative.dev sharedmain Package
 //
-func InitializeKafkaConfigMapWatcher(ctx context.Context, logger *zap.SugaredLogger, handler LoggingObserver, namespace string) error {
-
-	// Create A Watcher On The Configuration Settings ConfigMap & Dynamically Update Configuration
-	// Since this is designed to be called by the main() function, the default KNative package behavior here
-	// is a fatal exit if the watch cannot be set up.
-	watcher := sharedmain.SetupConfigMapWatchOrDie(ctx, logger)
-
+func InitializeKafkaConfigMapWatcher(ctx context.Context, watcher configmap.Watcher, logger *zap.SugaredLogger, handler LoggingObserver, namespace string) error {
```

Comment:
The start of the config map watcher is done by the caller (the owner of the watcher), not this method

Reply:
ok, gonna delete the `start` inside `InitializeKafkaConfigMapWatcher`.

thanks